include/sentry.h

```diff
@@ -1110,10 +1112,16 @@ SENTRY_API void sentry_user_consent_reset(void);
  */
 SENTRY_API sentry_user_consent_t sentry_user_consent_get(void);
 
+#ifdef SENTRY_PERFORMANCE_MONITORING
```

Comment:
ifdef-ing doc comments is pretty useless :-D Not even sure what it does.

Reply:
"please ignore this block if you are not using this compile flag" 😎